grails-app/views/admin/_permissionTablePaginated.gsp

```diff
@@ -0,0 +1,162 @@
+<r:require modules="jquery_bootstrap_datatable"/>
+
+<div class="pill-pane">
+    <div class="row well well-small" id="project-member-list">
+        <table style="width: 95%;" class="table table-striped table-bordered table-hover" id="member-list">
+            <thead>
+            <th>User Id</th>
+            <th>User Name</th>
+            <th>Role</th>
+            <th width="5%"></th>
+            </thead>
+            <tbody>
+            </tbody>
+        </table>
+    </div>
+    <div class="span5">
+        <div id="formStatus" class="hide alert alert-success">
+            <button class="close" onclick="$('.alert').fadeOut();" href="#">×</button>
+            <span></span>
+        </div>
+    </div>
+</div>
+
+<r:script type="text/javascript">
+    var table;
+
+    $(document).ready(function() {
+        table = $('#member-list').DataTable({
+            "bFilter": false,
+            "processing": true,
+            "serverSide": true,
+            "ajax": "${createLink(controller: 'project', action: 'getMembersForProjectIdPaginated')}/${project.projectId}",
```

Comment:
For consistency, can we move url's to top of the file.
Example: fcConfig. getMembersForProjectIdPaginatedUrl